BusBoard.ConsoleApp/Program.cs

```diff
@@ -30,19 +30,19 @@ private static void UserConsoleInteration()
         userInput = Console.ReadLine();
         if (userInput != "Exit")
         {
-          TflAPI TFLServerAPI = new TflAPI();
+          TflApi serverApi = new TflApi();
```

Comment:
Sorry if I wasn't clear enough. I think this should be `tflApi` as the variable name. I'm not sure what `server` corresponds to in this case.

So I think this line should be `TflApi tflApi = new TflApi();`